x/superfluid/proposal_handler.go

```diff
@@ -5,16 +5,18 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 
+	epochskeeper "github.com/osmosis-labs/osmosis/v7/x/epochs/keeper"
+
 	"github.com/osmosis-labs/osmosis/v7/x/superfluid/keeper"
 	"github.com/osmosis-labs/osmosis/v7/x/superfluid/keeper/gov"
 	"github.com/osmosis-labs/osmosis/v7/x/superfluid/types"
 )
 
-func NewSuperfluidProposalHandler(k keeper.Keeper) govtypes.Handler {
+func NewSuperfluidProposalHandler(k keeper.Keeper, ek epochskeeper.Keeper) govtypes.Handler {
```

Comment:
```suggestion
func NewSuperfluidProposalHandler(k keeper.Keeper, ek types.EpochKeeper) govtypes.Handler {
```

Change tot he expected keeper, rather than underlying struct.